fixtures/american-dj/mega-par-profile-plus

```diff
@@ -0,0 +1,990 @@
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [32, 63],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [64, 95],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [96, 127],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [128, 159],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [160, 191],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [192, 223],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "randomTiming": true
+        },
+        {
+          "dmxRange": [224, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        }
+      ]
+    },
+    "Program Selection Mode": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 51],
+          "type": "Maintenance",
+          "comment": "Dimming Mode",
+          "switchChannels": {
+            "Program Selection": "Color Macros",
+            "Program Speed / Sound Sensitivity": "Program Speed"
+          }
+        },
+        {
+          "dmxRange": [52, 102],
+          "type": "Maintenance",
+          "comment": "Color Macro Mode",
+          "switchChannels": {
+            "Program Selection": "Color Macros",
+            "Program Speed / Sound Sensitivity": "Program Speed"
+          }
+        },
+        {
+          "dmxRange": [103, 153],
+          "type": "Maintenance",
+          "comment": "Color Change Mode",
+          "switchChannels": {
+            "Program Selection": "Color Change Programs",
+            "Program Speed / Sound Sensitivity": "Program Speed"
+          }
+        },
+        {
+          "dmxRange": [154, 204],
+          "type": "Maintenance",
+          "comment": "Color Fade Mode",
+          "switchChannels": {
+            "Program Selection": "Color Fade Programs",
+            "Program Speed / Sound Sensitivity": "Program Speed"
+          }
+        },
+        {
+          "dmxRange": [205, 255],
+          "type": "Maintenance",
+          "comment": "Sound Active Mode",
+          "switchChannels": {
+            "Program Selection": "Sound Active Programs",
+            "Program Speed / Sound Sensitivity": "Sound Sensitivity"
+          }
+        }
+      ]
+    },
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "ColorPreset",
+          "colors": ["#000000"],
+          "comment": "Black"
```

Comment:
I would change this to a type of `NoFunction` so that it more reflects the Macros being off.
```suggestion
          "type": "NoFunction"
```